src/config.py

```diff
@@ -1,12 +1,7 @@
-credentials = {
-    'username' : 'xxxx',
-    'password' : 'xxxx',
-    'emailID' : 'xxxx',
-    'emailPassword' : 'xxxx',
-    'database' : 'xxxx',
-    'token': 'xxxx'
-}
+
 emojiLocal = {
 'ACCEPT' : "\U00002705",
-'DENY' : "\U0000274e"
+'DENY' : "\U0000274e",
+'STONKSUP': "<:Stonks:832284727578460252>",
```

Comment:
I think this is our private repo emoji ID . 
It will not work on the purdue or other university servers.
Can this be replaced with something generic ? say a thumbs up and thumbs down?